e3sm_diags/driver/utils/dataset_xr.py

```diff
@@ -459,7 +444,16 @@ def _open_climo_dataset(self, filepath: str) -> xr.Dataset:
         # No need to decode times because the climatology is already calculated.
         # Times only need to be decoded if climatology is being calculated
         # (time series datasets).
-        args = {"path": filepath, "decode_times": False, "add_bounds": ["X", "Y"]}
+        # NOTE: This GitHub issue explains why the "coords" and "compat" args
+        # are defined as they are below: https://github.com/xCDAT/xcdat/issues/641
+        args = {
+            "paths": filepath,
+            "decode_times": False,
+            "add_bounds": ["X", "Y"],
+            "coords": "minimal",
+            "compat": "override",
+            "chunks": "auto",
+        }
```

Comment:
Notable change. I am going to remove `"chunks": "auto"` because we end up loading the dataset into memory. This means downstream computational operations are all serial within the single process.